src/platform_impl/linux/wayland/seat/pointer/mod.rs

```diff
@@ -109,9 +109,10 @@ impl WinitPointer {
         let serial = Some(self.latest_serial.get());
         for cursor in cursors {
             if self.pointer.set_cursor(cursor, serial).is_ok() {
-                break;
+                return;
             }
         }
+        warn!("Failed to set cursor");
```

Comment:
I don't think it's informative enough, you should provide the thing you've queried for, e.g. `cursor_icon`.

Reply:
I have made this change.